### ISSUE TITLE
Does only http_client support 0-rtt? echo_client -0 invalid parameter？

### ISSUE THREAD

**Issue by muyiwq** [litespeedtech/lsquic#270]
How to enable 0-rtt for quic in short link scenarios, ./echo_client -s 127.1:1234 0 test.txt, add the 0 parameter without-to the client, and -0 reports invalid option, and the client connects several times and captures packets but it is not 0-rtt, the client shares the same engine, how to establish multiple quic connections, will there be 0-rtt only when the same engine connects to the same server and it is not the first connection? Any suggestions for me? Looking forward to your reply. .

**Comment by muyiwq** [comment]
Currently only http_client supports -0 <file> 0-rtt? Why can't echo_client or non-http client modified based on echo_client use -0? I checked the code because the getopt() function parameter __shortopts does not include the -0 parameter. If I modify this, can the non-http client support 0-rtt? I am still not 0-rtt after capturing the packet after modification? Thanks in advance。。。

**Comment by muyiwq** [comment]
Currently only http_client supports -0 0-rtt? Why can't echo_client or non-http client modified based on echo_client use -0? ​I checked the code because the getopt() function parameter __shortopts does not include the -0 parameter. 

If I modify this, can the non-http client support 0-rtt? I am still not 0-rtt after capturing the packet after modification，Is it caused by my wrong modification?
How to get the sess_resume parameter in the lsquic_engine_connect() function from the first connection in non-http client? 
Does the parameter of sess_resume has value for the first time mean that the next connection is 0-rtt

**Comment by muyiwq** [comment]
Currently only http_client supports -0 0-rtt? Why can't echo_client or non-http client modified based on echo_client use -0? ​I checked the code because the getopt() function parameter __shortopts does not include the -0 parameter. 

If I modify this, can the non-http client support 0-rtt? I am still not 0-rtt after capturing the packet after modification，Is it caused by my wrong modification?
How to get the sess_resume parameter in the lsquic_engine_connect() function from the first connection in non-http client? 
Does the parameter of sess_resume has value for the first time mean that the next connection is 0-rtt？

**Comment by litespeedtech** [comment]
0-rtt client has been implemented for gQUIC only, it has not been implemented for HTTP/3 client yet. Pull requests are welcome.

**Comment by muyiwq** [comment]
Thank you for your reply, how to use 0-rtt in gquic client？Specify the version of gquic, do you need to specify the -0 parameter? echo_client.c does not support the -0 parameter,  How to get the sess_resume parameter in the lsquic_engine_connect() function from the first connection？

**Comment by litespeedtech** [comment]
Yeah, you just add parameter to use gQUIC protocol instead of h3-xx . 
echo_client is not a full featured client, just an example, use http_client for reference.

**Comment by muyiwq** [comment]
I got it，I will try it client gquic 0-rtt， thank you。。
